auth/presentation/src/main/java/com/nibalk/tasky/auth/presentation/components/AuthClickableText.kt

```diff
@@ -0,0 +1,80 @@
+package com.nibalk.tasky.auth.presentation.components
+
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+import com.nibalk.tasky.core.presentation.themes.Inter
+import com.nibalk.tasky.core.presentation.themes.TaskyDarkBlue
+import com.nibalk.tasky.core.presentation.themes.TaskyGray
+import com.nibalk.tasky.core.presentation.themes.TaskyTheme
+
+@Composable
+fun AuthClickableText(
+    isVisible: Boolean,
```

Comment:
Such isVisible booleans are quite unnecessary to pass to sub-composables. Maybe for things like dialogs which are always hidden at some point, but for normal composables I'd just leave that parameter and let the parent decide when it's visible or not (the `if(isVisible)` can of course just be put in the parent as well)